docs/tracking/google-tag-manager.md

```diff
@@ -29,11 +34,19 @@ const sdk = createInstance({
   ) {
     // push to dataLayer here
     window.dataLayer.push({
-      event: 'featurevisor-activation',
+      event: 'featureVisorActivation',
       featureKey,
       variationValue,
       capturedAttributes,
     });
   }
 })
 ```
+
+## Pass the experiment name and its metadata as a Custom Dimension in your GTM Tags
+
+To be able to filter your data and create segmentation for each of your experiments, make sure to pass the unique id of the experiment and its variation as a Custom Dimension in your GA4 tags. To achieve this, make sure to first register a new Custom Dimension on your interface and name it `fv_experiment_id`. You can decide whether to make that Custom Dimension an Event Paramenter or a User Property based on the principles of your custom analytics implementation. Once done, add the parameter to all of your GA4 tags in Google Tag Manager to attach the information to the event.
```

Comment:
`unique id` => `unique ID`

---

worth having a link for "Custom Dimension" text?

we can also briefly explain what a Custom Dimension is, and how it can be named giving readers an option.

links in markdown can be written as:

```
My sentence with a [link](https://google.com) here.
```

Reply:
done :)